packages/api/lib/reconciliationReport.js

```diff
@@ -117,6 +117,30 @@ function convertToDBGranuleSearchParams(params) {
   return removeNilProperties(searchParams);
 }
 
+/**
+ *
+ * @param {Object} params - request params to convert to database params
+ * @returns {Object} object of desired parameters formated for database granule search
+ */
+function convertToDBScanGranuleSearchParams(params) {
+  const {
+    collectionId,
+    granuleId,
```

Comment:
Is the `granuleId` from the `search` box in dashboard granules page?  The search field is actually `infix`, matches part of the granuleId.

Reply:
nvm.  I see `contains` is used for the db search.